assets/stylesheets/overall.scss

```diff
@@ -33,8 +33,7 @@ body {
     position: absolute;
     bottom: 0;
     width: 100%;
-    // set the fixed height of the footer here
-    height: 30px;
+    padding-bottom: 10px;
```

Comment:
For me the result is worse than before.

Before:

![image](https://github.com/os-autoinst/openQA/assets/1204189/2c5ae0b3-7a53-49ce-a21b-0ff38b97858d)


After:

![image](https://github.com/os-autoinst/openQA/assets/1204189/6eaf71dc-671a-49f6-804a-f6d5261c9a2d)

Reply:
it seems you want to address a quite different problem. Your screenshots look like a tiny resolution.

If this is about the footer hiding the button we could give the footer a `z-index: -1;` so the footer would overlap other content on the screen

Reply:
Yes, I also thought about using z-index. so the footer would **not** overlap content.

Reply:
> Your screenshots look like a tiny resolution.

Well, let's say the *effective* width is very small. And I'd argue that this use case is not very important because with such a small width the rest of the page is not very usable anymore anyway. So I'd still be fine with this change.

> If this is about the footer hiding the button we could give the footer a `z-index: -1;` so the footer would overlap other content on the screen

You can try that.

Reply:
ok. I added another commit with `z-index: -1;` which fixes the issue that @kalikiana pointed out at least in my tests. Acceptable?

Reply:
As my previous message, and in particular the last screenshot, was supposed to clarify: **I don't see any issue besides the background color unless I make the page tiny**. The z-index just breaks it differently by hiding the footer.

You can argue that you don't care about browser other than whatever you tested. For me all of these suggestions worsen the situation.